--- a/dist/index.js
+++ b/dist/index.js
@@ -78,10 +78,6 @@
             ...Object.keys(b).length > 0 ? c ? { transform: { inject: b } } : { plugins: [x(b)] } : {}
           }
         },
-        esbuild: {
-          // In dev, the global polyfills need to be injected as a banner in order for isolated scripts (such as Vue SFCs) to have access to them.
-          banner: i ? a : void 0
-        },
         optimizeDeps: {
           exclude: [
             ...u
